graal/graal_bindist.bzl

```diff
@@ -209,6 +210,11 @@ def _get_platform(ctx):
         return "linux-%s" % arch
     elif ctx.os.name == "mac os x":
         if arch == "arm64" or arch == "aarch64":
+            if ctx.attr.version != "22.1.0":
```

Comment:
probably want to swap this for a safe split with a major/minor check eventually, but since `bindist` is hardcoded rn anyway, i figured it's all the same until the next release.

Reply:
oh, good catch... :/ shame we don't have a good CI

😬

Reply:
yeah, it would be better to have a better approach, but good to at least fix the issue now.